thumbnail/thumbnail_magick.go

```diff
@@ -176,6 +176,7 @@ const (
 	FORMAT_PNG   = iota
 	FORMAT_WEBP  = iota
 	FORMAT_BMP   = iota
+	FORMAT_HEIC  = iota
```

Comment:
@yoya san. The 'q' (quality) parameter seem not working for HEIF format. Could you help to check?

Reply:
I've confirmed the defect, working on it. wait a moment.
SetImageCompressionQuality works good for JPEG but seems bad for HEIC.
There may be a problem with the passing quality from MagickWand to MagickCore.

Reply:
@yoya san, I create a issue https://github.com/smartnews/yoya-thumber/issues/27 to track. And the other parts look good to me. I will merge this pull request while you can working on that issue.